### PR TITLE
add contribution guideline

### PR DESCRIPTION
**Pull request by itaysk**
Once this is approved I'll add to funnel and vuln-list-update as well

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+1. Your PR is more likely to be accepted if it includes tests (We have not historically been very strict about tests, but we would like to improve this!).
+1. If your PR affects the user experience in some way, please update the Readme and the cli help accordingly.
+
+## Understand where your pull request belongs
+
+Trivy is composed of several different repositories that work togeather:
+- [Trivy](https://github.com/aquasecurity/trivy) is the client side, user facing, command line tool.
+- [vuln-list](https://github.com/aquasecurity/vuln-list) is a vulnerabilities database, aggregated from different sources, and normalized for easy consumption. This of this as the "server" side of the trivy command line tool. **There should be no pull requests to this repo** 
+- [vuln-list-update](https://github.com/aquasecurity/vuln-list-update) is the code that maintains the vuln-list database.
+- [funnel](https://github.com/aquasecurity/funnel) is a library for extracting system information containers. It is being used by trivy to find testable subjects in the container image.
```
Comment (knqyf263) [inline CONTRIBUTING.md]:
```suggestion
- [fanal](https://github.com/aquasecurity/fanal) is a library for extracting system information containers. It is being used by trivy to find testable subjects in the container image.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+1. Please add the associated Issue in the PR description.
+1. There's no need to add or tag reviewers.
+1. If a reviewer commented on your code, or asked for changes, please remember to mark the discussion as resolved after you address it. PRs with unresolved issues should not be merged (even if the comment is unclear or requires no action from your side).
+1. Please include a comment with the results before and after your change.
+1. Your PR is more likely to be accepted if it includes tests (We have not historically been very strict about tests, but we would like to improve this!).
+1. If your PR affects the user experience in some way, please update the Readme and the cli help accordingly.
+
+## Understand where your pull request belongs
+
+Trivy is composed of several different repositories that work togeather:
```
Comment (knqyf263) [inline CONTRIBUTING.md]:
```suggestion
Trivy is composed of several different repositories that work together:
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+Thank you for taking interest in contributing to Trivy !
+
+## Issues
+- Feel free to open issues for any reason. When you open a new issue you'll have to select a the issue kind: bug/feature/support and fill the required information based on the selected template.
```
Comment (knqyf263) [inline CONTRIBUTING.md]:
```suggestion
- Feel free to open issues for any reason. When you open a new issue, you'll have to select an issue kind: bug/feature/support and fill the required information based on the selected template.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+1. Every Pull Request should have an associated bug or feature issue, unless you are fixing a trivial documentation issue.
+1. Your PR is more likely to be accepted if it focuses on just one change.
+1. Describe what the PR does. There's no convention enforced, but please try to be concise and descriptive. Treat the PR description as a commit message. Titles that starts with "fix"/"add"/"improve"/"remove" are good examples.
+1. Please add the associated Issue in the PR description.
+1. There's no need to add or tag reviewers.
+1. If a reviewer commented on your code, or asked for changes, please remember to mark the discussion as resolved after you address it. PRs with unresolved issues should not be merged (even if the comment is unclear or requires no action from your side).
+1. Please include a comment with the results before and after your change.
+1. Your PR is more likely to be accepted if it includes tests (We have not historically been very strict about tests, but we would like to improve this!).
+1. If your PR affects the user experience in some way, please update the Readme and the cli help accordingly.
```
Comment (knqyf263) [inline CONTRIBUTING.md]:
```suggestion
1. If your PR affects the user experience in some way, please update the README and the cli help accordingly.
```

**Comment by itaysk** [comment]
@knqyf263 you have any idea why the CLA check is not fulfilled?

**Comment by knqyf263** [comment]
I'm just wondering...

**Comment by knqyf263** [comment]
I re-ran CLA check.
https://cla-assistant.io

**Pull request merged**